client/src/components/shared/ModalOnboarding/ModalOnboarding.tsx

```diff
@@ -66,8 +66,7 @@ const ModalOnboarding = (): ReactNode => {
     steps: stepsToUse,
   });
 
-  const currentStep =
-    !isFetchingUserTOS && stepsToUse.length > 0 ? stepsToUse[currentStepIndex] : null;
+  const currentStep = stepsToUse.length > 0 ? stepsToUse[currentStepIndex] : null;
```

Comment:
Please expand to e.g.
```suggestion
  // For multisig users we refetch ToS in a setInternval, so isFetching here causes loop refreshes.
  const currentStep = !isFetchingUserTOS && !isContract && stepsToUse.length > 0 ? stepsToUse[currentStepIndex] : null;
```